src/components/Back.tsx

```diff
@@ -12,7 +11,7 @@ export const Back: FC<Props> = ({ to }) => {
 
   return (
     <Icon
-      icon={faChevronLeft}
+      icon="chevron-left"
```

Comment:
This won't work because we need an icon library.
![image](https://user-images.githubusercontent.com/52031625/95024285-57979880-0682-11eb-9c0a-de7835243c37.png)
You can find more info here: https://www.digitalocean.com/community/tutorials/how-to-use-font-awesome-5-with-react

Reply:
Right! I'll fix it right away